emailproxy.py

```diff
@@ -710,7 +720,6 @@ def get_account_with_catch_all_fallback(option):
                 if username not in config.sections():
                     config.add_section(username)  # in catch-all mode the section may not yet exist
                     REQUEST_QUEUE.put(MENU_UPDATE)  # make sure the menu shows the newly-added account
-                config.set(username, 'token_salt', token_salt)
```

Comment:
I think a tuple is the best choice, because to fix the `KeyError` we'd need to create a new section, and although the config is only saved on a successful login, a later save (for example on a successful login with a different account) would include this new section. We could of course detect that and ignore/remove empty sections, but I think returning a tuple from the new function is the simplest approach here.